faker/sphinx/documentor.py

```diff
@@ -131,8 +128,7 @@ def _write_localized_provider_index():
 def _write_localized_provider_docs():
     for locale in AVAILABLE_LOCALES:
         info = _get_localized_provider_info(locale)
-        fname = os.path.join(DOCS_ROOT, 'locales', '{}.rst'.format(locale))
-        with open(fname, 'wb') as fh:
+        with (DOCS_ROOT / 'locales' / '{}.rst'.format(locale)).open('wb') as fh:
```

Comment:
we could use an f-string here too:
```suggestion
        with (DOCS_ROOT / 'locales' / f'{locale}.rst').open('wb') as fh:
```

Reply:
ditto